api/v1/metric_jsonapi.go

```diff
@@ -0,0 +1,82 @@
+/*
+ *
+ * k6 - a next-generation load testing tool
+ * Copyright (C) 2021 Load Impact
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package v1
+
+import (
+	"time"
+
+	"go.k6.io/k6/stats"
+)
+
+type metricJSONAPI struct {
+	Data metricData `json:"data"`
+}
+
+// MetricsJSONAPI is JSON API envelop for metrics
+type MetricsJSONAPI struct {
+	Data []metricData `json:"data"`
+}
```

Comment:
Why `Metrics` is exported and `metric` not?

Reply:
A `MetricsJSONAPI` is used outside the package, and `metricJSONAPI` is not. So I decided to keep it unexported to limit usages.